core/src/main/scala/com/cloudera/livy/sessions/SessionState.scala

```diff
@@ -61,19 +61,19 @@ object SessionState {
     override def toString = "shutting_down"
   }
 
-  case class Error(time: Long = System.currentTimeMillis()) extends SessionState {
+  case class Error(time: Long = System.nanoTime()) extends SessionState {
```

Comment:
The `SessionState` itself is, but the timestamp is not. The `/:id/state` API will return the state's string form, which is basically "started", "dead" etc.